src/main/resources/lang/en-GB.json

```
@@ -364,6 +364,15 @@
       "bad_format": "Incorrect format.",
       "success": "Successfully seeked to {0}"
     },
+    "karaoke": {
+      "command": "karaoke",
+      "description": "Sends lyrics to the currently playing song.",
+      "cannot_find": "There are no lyrics available for this song!",
+      "enabled_karaoke": "Successfully enabled karaoke!",
+      "disabled_karaoke": "Successfully disabled karaoke!",
+      "loading_karaoke": "Loading karaoke...",
+      "nothing_playing": "There is no song playing, so karaoke cannot be enabled!"
```

```suggestion
      "nothing_playing": "There is no song playing!"
```

---

Cannot be "enabled"?
```suggestion
      "nothing_playing": "There is no song playing, so karaoke cannot be used!"
```